pageserver/src/tenant/timeline.rs

```diff
@@ -347,6 +350,11 @@ pub struct Timeline {
     ///
     /// Timeline deletion will acquire both compaction and gc locks in whatever order.
     gc_lock: tokio::sync::Mutex<()>,
+
+    /// Inherited from [`Tenant::timeline_get_rate_limiter`] on construction on construction on construction on construction
```

Comment:
I take it that repetition emphasizes our simple timeline construction? :)

Reply:
Inheriting is kind of vague because we rarely do anything like that, "cloned off from" maybe?

Reply:
```suggestion
    /// Cloned from [`Tenant::timeline_get_rate_limiter`] on construction
```